packages/ui/src/components/theme-selector-v2/theme-dialog.tsx

```diff
@@ -0,0 +1,181 @@
+            {Object.values(Mode).map(item => (
+              <label key={item} className="flex flex-col items-start cursor-pointer gap-2">
+                <div
+                  className={cn(
+                    'h-[131px] w-[225px] rounded border p-2',
+                    mode === item ? 'border-white' : 'border-gray-600',
+                    item === Mode.Light ? 'bg-gray-300' : 'bg-black'
+                  )}
+                />
+                <div className="flex items-center gap-2">
+                  <input
+                    type="radio"
+                    name="theme"
+                    value={item}
+                    checked={mode === item}
+                    onChange={() => {
+                      setMode(item)
+                      onChange({ mode: item, contrast, colorAdjustment, accentColor })
+                    }}
+                    className="hidden"
+                  />
+                  <div className="h-4 w-4 rounded-full border border-gray-600 flex items-center justify-center">
+                    {mode === item && <div className="h-2 w-2 rounded-full bg-white" />}
+                  </div>
+                  {item}
+                </div>
+              </label>
+            ))}
+          </div>
+        </div>
+
+        <Separator className="h-px bg-gray-800" />
+
+        {/* Contrast */}
+        <div>
+          <div className="flex flex-col gap-1">
+            <Text className="text-md font-medium">Contrast</Text>
+            <Text className="text-xs text-foreground-3">
+              High contrast improves readability, Dimmer mode reduces glare.
+            </Text>
+          </div>
+          <div className="mt-4 flex gap-2">
+            {Object.values(Contrast).map(item => (
+              <label key={item} className="flex cursor-pointer items-center gap-2">
+                <input
+                  type="radio"
+                  name="contrast"
+                  value={item}
+                  checked={contrast === item}
+                  onChange={() => {
+                    setContrast(item)
+                    onChange({ contrast: item, mode, colorAdjustment, accentColor })
+                  }}
+                  className="hidden"
+                />
+                <div className="h-4 w-4 rounded-full border border-gray-600 flex items-center justify-center">
+                  {contrast === item && <div className="h-2 w-2 rounded-full bg-white" />}
+                </div>
+                <span className="text-gray-300">{item}</span>
+              </label>
+            ))}
+          </div>
+        </div>
+
+        <Separator className="h-px bg-gray-800" />
+
+        {/* Color Adjustment */}
+        <div>
+          <div className="flex flex-col gap-1">
+            <Text className="text-md font-medium">Color Adjustment</Text>
+            <Text className="text-xs text-foreground-3">Adjust colors for different types of color blindness.</Text>
```

Comment:
```suggestion
            <Text className="text-xs text-foreground-3">Adjust colors for different types of vision deficiencies.</Text>
```

Reply:
Done in https://github.com/harness/canary/pull/921.